### ISSUE TITLE
Unable to save changed location for existing meeting

### ISSUE THREAD

**Issue by wombleysclapboards** [code4recovery/12-step-meeting-list#137]
**Describe the bug**
When I try to change the location for an existing meeting, publishing the edited meeting causes the location to revert back to the previous - new location is lost.
**To Reproduce**
1. Meetings > select any existing record
2. Click on 'Edit'
3. Scroll down to 'Location', change entry to another location already in the database
4. Click 'Update'
5. 'Location' not saved - reverts back to original

**Expected behavior**
Revised location should be saved.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS 10.15.6
 - Browser [e.g. chrome, safari] Vivaldi (Chrome engine) 3.1.1929.48
 - Version [e.g. 22] 3.1.1929.48

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] N/A - not phone related

**Additional context**
Problem occurs with multiple records and has been experienced by another admin using MS Win10/Chrome

**Comment by tech2serve** [comment]
@wombleysclapboards 

What version of the plugin are you using? Most location related issues occur due to a problem with the address geolocating (i.e., Google is not returning a valid address). Can you share a couple of the addresses you're trying to enter? 

And the link to your meeting list, please?

**Comment by wombleysclapboards** [comment]
Sorry for the lapse in time, I took a bit of time off. Plugin version is 3.7.2. Example of a problem meeting: https://www.pdxaa.org/index.php/meetings/nite-owls-7/
Trying to change address from: Portland Alano Club, 909 NW 24th Ave, Portland, OR 97210, USA
To: 4th Dimension Recovery Center, 3807 NE Martin Luther King Jr Blvd, Portland, OR 97212, USA
Both locations appear to geolocate - maps appear to be correct; these are not the only meetings or locations where this problem has occurred.
Link to Meetings list: https://www.pdxaa.org/index.php/meetings/
Thanks!

**Comment by tech2serve** [comment]
Not sure if this is related, but when I follow the link to the Nite Owls meeting, I get this address:

<img width="950" alt="image" src="https://user-images.githubusercontent.com/31904316/91592808-586b3b00-e91c-11ea-94fa-d145f6e91224.png">

If that isn't what you're getting, something even weirder is going on :-).

Let's start with having you clear the cache in your browser and for the website if possible. 

When you try to change the address, are you deleting both of the location and address lines before entering the new information?

**Comment by wombleysclapboards** [comment]
The Hillsboro location was in there because I tried changing to a different location so see whether it was just the one 4th Dimension Recovery Center that was causing problems. This a daily meeting (seven instances) and I found that the new location was showing up on the public side (a good thing) but not in the editing screen so I tried refreshing permalinks, no help; clearing browser cache, no help; clearing website cache - aha! - that did it. Thanks!
